static/src/components/builder/components/TaskPresenter/TaskPresenterForm.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div class="row">
+    <h4>Task Presenter Settings</h4>
+    <div class="form-row">
+      <p>Saving partial work functions for all schedulers.</p>
+      <input
+        id="allow-save-work"
+        v-model="allowSaveWork"
+        :checked="allowSaveWork"
+        type="checkbox"
+      >
+      <label
+        for="allow-save-work"
+      >
+        &ensp;Allow partial work to be saved
+      </label>
+    </div>
+    <div v-if="allowSaveWork">
+      <label
+        for="auto-save-seconds"
+        style="font-weight: lighter"
+      >
+        &emsp;&ensp;Seconds (min.5) between auto saves&emsp;
```

Comment:
Should we remove `(min.5)` from this text, and instead just allow validation to display the message? It may add confusion in this display. Thank you.

Reply:
Done